BraintreeModels/src/main/java/com/braintreepayments/api/PaymentMethodType.java

```diff
@@ -16,7 +16,8 @@
         PaymentMethodType.VENMO,
         PaymentMethodType.VISA_CHECKOUT,
         PaymentMethodType.GOOGLE_PAY,
-        PaymentMethodType.LOCAL_PAYMENT
+        PaymentMethodType.LOCAL_PAYMENT,
+        PaymentMethodType.PAYPAL_NATIVE
```

Comment:
I'm not sure that this is necessary. These types are used to differentiate PaymentMethodNonce` types, but it looks like we are returning a `PayPalAccountNonce` at the end of this flow (the same nonce type as the legacy PayPal flow), so I don't see a need to differentiate types here.